package/kernel/linux/modules/netdevices.mk

```diff
@@ -539,15 +539,15 @@ endef
 
 $(eval $(call KernelPackage,8139cp))
 
-
 define KernelPackage/r8169
   SUBMENU:=$(NETWORK_DEVICES_MENU)
   TITLE:=RealTek RTL-8169 PCI Gigabit Ethernet Adapter kernel support
   DEPENDS:=@PCI_SUPPORT +kmod-mii +r8169-firmware +kmod-phy-realtek
   KCONFIG:=CONFIG_R8169 \
     CONFIG_R8169_NAPI=y \
     CONFIG_R8169_VLAN=n
-  FILES:=$(LINUX_DIR)/drivers/net/ethernet/realtek/r8169.ko
+  FILES:=$(LINUX_DIR)/drivers/net/ethernet/realtek/r8169.ko \
+	  $(LINUX_DIR)/drivers/net/phy/mdio_devres.ko
```

Comment:
Please make hanging indent consistent, at least locally.
Shouldn't this get a kernel switch?